brave/brave-core
# Adds unique constraint on unblinded_tokens table

Resolves https://github.com/brave/brave-browser/issues/9889

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- start from master
- enable rewards 
- claim grant
- close the browser 
- open db and duplicate some unblinded tokens
- start the browser and make sure that balance went up based on duplication
- close the browser
 - switch to this PR
- start the browser
- make sure that balance is now back to the original grant value

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
